webservices/tests/test_integration.py

```diff
@@ -0,0 +1,17 @@
+import manage
+from webservices.rest import db
+from webservices.tests import common
+
+
+class TestViews(common.IntegrationTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestViews, cls).setUpClass()
+        manage.update_schemas(processes=1)
+
+    def test_update_schemas(self):
```

Comment:
Seemed like an opportune time to sneak in an extra test. This will import the test subset, build the materialized views, and verify that the views have been populated (or at least contain more than zero rows). Practically, this means that the build will fail if anybody checks in a SQL script that doesn't run.

@arowla @LindsayYoung this should make it harder for me to check in SQL errors :smiley: